File: weeak-1/sumTwoNumbers.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter two numbers ", function(twoNumbers) {
   twoNumbers = twoNumbers.split(" ")
   const num1 = parseInt(twoNumbers[0]) 
   const num2 = parseInt(twoNumbers[1]) 

   const sum = num1+num2
   console.log(`sum of two numbers is ${sum}`)
});

rl.on("close", function() {
    console.log("\nBYE BYE !!!");
    process.exit(0);
});

File: weeak-1/sumTwoNumbers/sumTwoNumbers.go
package main
import(
	"fmt"
	 "bufio"
	 "os"
	 "strconv"
	 "strings"
)

func main(){
	scan := bufio.NewScanner(os.Stdin)
	scan.Scan()
	text := string(scan.Text())
	numbers := strings.Split(text," ")
	num1, _ := strconv.Atoi(numbers[0])
	num2, _ := strconv.Atoi(numbers[1])
	sum := num1+num2
	fmt.Println(sum)
}

